src/rules/requireHyphenBeforeParamDescription.js

```diff
@@ -47,8 +48,28 @@ export default iterateJsdoc(({
   };
 
   utils.forEachPreferredTag('param', checkHyphens);
-  if (checkProperties) {
-    utils.forEachPreferredTag('property', checkHyphens);
+  if (tags) {
+    const tagEntries = Object.entries(tags);
+    tagEntries.forEach(([tagName, circumstance]) => {
+      if (tagName === '*') {
+        const preferredParamTag = utils.getPreferredTagName({tagName: 'param'});
+        (jsdoc.tags || []).forEach(({tag}) => {
+          if (tag === preferredParamTag || tagEntries.some(([tagNme]) => {
+            return tagNme !== '*' && tagNme === tag;
```

Comment:
Should this be `tagName` as opposed to `tagNme`?

Reply:
The `tagNme` is just used because earlier in this block, there is a different `tagName` (and there is a linting rule discouraging shadowing, i.e., reusing the same name for a different variable in a more nested scope).